### ISSUE TITLE
Bug: load-list checks aren't reported when using `magik-lint.jar --show-checks`

### ISSUE THREAD

**Issue by sebastiaanspeck** [StevenLooman/sonar-magik#646]
`MagikLint.getAllCheckClasses()` is missing `LoadListCheckList`:

https://github.com/StevenLooman/magik-tools/blob/a0af5761d7f44ea6cb6b620721d2f0f32d5be68b/magik-lint/src/main/java/nl/ramsolutions/sw/magik/lint/MagikLint.java#L184-L192

```diff
    return Stream.of(
            ProductDefCheckList.INSTANCE.getBaseChecks().stream(),
            ModuleDefCheckList.INSTANCE.getBaseChecks().stream(),
+           LoadListCheckList.INSTANCE.getBaseChecks().stream(),
            MagikCheckList.INSTANCE.getBaseChecks().stream())
```

When using `magik-lint.jar --show-checks`, the load-list checks aren't reported (`load-list-entry-exists` and `load-list-syntax-error`).

**Comment by StevenLooman** [comment]
Thanks!
